framework/sac_rlhf.py

```diff
@@ -78,6 +78,19 @@ class Args:
     teacher_learning_rate: float = 1e-3
     """the learning rate of the teacher"""
 
+    # Simulated Teacher
+    teacher_sim_beta: float = -1
```

Comment:
According to the paper, the teacher becomes a perfectly rational and deterministic as β →∞, whereas β = 0 produces uniformly random choices. Why did you pick -1 as a value? Is -1 even a valid value in the value range?

Reply:
While a large positive beta also creates deterministic behavior, using beta = -1 is a convention to ensure the oracle has a standardized configuration distinct from noisy or stochastic teachers.